highfidelity/hifi
# 4684 Add collapse button on the virtual keyboard

*** Test plan ***: 

1. Switch to VR
2. Open 'go to location', select input field so that virtual keyboard appear. 
3. Ensure virtual keyboard has new 'collapse' button like on the picture below:

![image](https://user-images.githubusercontent.com/23638/32145894-c96586b4-bce0-11e7-8530-8a7c84c8bb8d.png)

4. Ensure clicking on 'collapse' button collapses keyboard in the same way as clicking outside the keyboard.
5. Ensure re-selecting input field after collapsing keyboard still raises keyboard. 

Note: It turned out that some dialogs are implemented in a bit different way than others, so please check as much different screens with virtual keyboard as possible and let me know what screen causes issues (hopefully no one, but still..)
